kurtosis-module/impl/participant_network/cl/availability_waiter/availability_waiter.go

```diff
@@ -7,11 +7,17 @@ import (
 	"time"
 )
 
+//READY state means that the Beacon node is synced
+//the eth beacon endpoint /eth/v1/node/health has returned 200 status code, see more here: https://ethereum.github.io/beacon-APIs/#/Node/getHealth
+const waitForAvailabilityExpectedStatus = "READY"
+
 func WaitForBeaconClientAvailability(restClient *cl_client_rest_client.CLClientRESTClient, numRetries uint32, timeBetweenRetries time.Duration) error {
 	for i := uint32(0); i < numRetries; i++ {
-		_, err := restClient.GetHealth()
+		status, err := restClient.GetHealth()
 		if err == nil {
-			// TODO check the healthstatus???
+			if status == waitForAvailabilityExpectedStatus {
+				return nil
+			}
 			return nil
```

Comment:
Wait doesn't this being here mean that the check above doesn't matter, since this will always return `nil` if err == nil??

Reply:
uuuh my mistake, I fill fix it